pkgs/stdenv/generic/check-meta.nix

```diff
@@ -117,6 +116,16 @@ let
   showLicense = showLicenseOrSourceType;
   showSourceType = showLicenseOrSourceType;
 
+  hasWarning = attrs: (attrs.meta.issues or []) != [];
+
+  ignoreWarningsDefaultPredicate = x: builtins.elem (getName x) (config.ignoreWarningsPackages or []);
+  ignoreWarningsPredicate = x: (config.ignoreWarningsPredicate or ignoreWarningsDefaultPredicate) x;
+
+  hasIgnoredWarning = attrs:
+    !(hasWarning attrs) ||
+    ignoreWarningsPredicate attrs ||
```

Comment:
I'd like to add a trace-level warning for when `!hasWarnings &&  ignoreWarningsPredicate` to help people keep their allow list clean. However, this would mean that `ignoreWarningsPredicate` gets evaluated for every single derivation (compared to only `hasWarning` at the moment). Should I be concerned about performance here?